app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    rating_list = []
+    for player in players:
+        rating_list.append(player.get_rating())
+    return sum(rating_list)
```

Comment:
The creation of `rating_list` is unnecessary. You can directly use a generator expression inside the `sum` function to improve efficiency. According to the checklist, avoid creating redundant lists. Consider using `return sum(player.get_rating() for player in players)` instead.